src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -415,6 +438,13 @@ def get_command_table(module_name=None):
         logger.debug("Loaded all modules in %.3f seconds. "
                      "(note: there's always an overhead with the first module loaded)",
                      cumulative_elapsed_time)
+    try:
+        # We always load extensions even if the appropriate module has been loaded
+        # as an extension could override the commands already loaded.
```

Comment:
In one of the docs, I mention "Currently, there is no repository service or marketplace that you publish your extension to."